apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/[keyId]/page.tsx

```diff
@@ -276,8 +276,10 @@ export default async function APIKeyDetailPage(props: {
             </EmptyPlaceholder.Description>
           </EmptyPlaceholder>
         )}
+        <h2 className="text-2xl font-semibold leading-none tracking-tight mt-8">
+          Latest Verifications
+        </h2>
```

Comment:
huh, why above the divider?

Reply:
Other way around looks bad IMO. 
![image](https://github.com/user-attachments/assets/340e7303-e15e-4e40-96fc-4aa1279139b7)
Like this, but we can roll it back if you want.